### ISSUE TITLE
Remove the cooldown from Spy sapper on regular buildings (but not robots) 

### ISSUE THREAD

**Issue by Thestickman391** [mastercomfig/team-comtress-2#197]
Just opening an issue to request for comment. This seems unintentional to me and it makes it very hard to deal with engibot nests as Spy.

**Comment by wgetJane** [comment]
how can you determine that this is unintentional?

**Comment by maximsmol** [comment]
This will most likely be wontfix no matter what, since this is a big balance change

**Comment by wgetJane** [comment]
> This will most likely be wontfix no matter what, since this is a big balance change

i'm not too sure about that though

melee attacks being blocked by teammates seem to be intentional, but a fix was implemented anyway since it *feels* unintentional
and it also has balance implications for classes like demoknight and spy

**Comment by maximsmol** [comment]
Yeah but I guess the melee attacks thing falls under the "'features' that everyone hates" category + precedent from RGL running a plugin with melee passing through teammates

**Comment by Thestickman391** [comment]
Nobody likes the fact that spy's sapper gets put on cooldown after sapping regular buildings in MvM...

**Comment by Yttrium-tYcLief** [comment]
> Nobody likes the fact that spy's sapper gets put on cooldown after sapping regular buildings in MvM...

While true, and definitely bad game design that I would personally fix, it still falls outside the scope of this project.
